docs/api/components/schemas.yaml

```diff
@@ -707,13 +732,11 @@ Retrieved-replication:
       default: false
     enable_delta_sync:
       description: |-
-        This will turn on delta- sync for the replication. This works in conjunction with the database level setting `delta_sync.enabled`
+        If the field is omitted, this will use the delta-sync setting of the database configuration `delta_sync.enabled`.
```

Comment:
Is that actually true? My reading of the code matches the old description. An omitted value means `false`/turn off delta sync.

Reply:
I read this like this:

Blip delta sync is set on initialization: 
https://github.com/couchbase/sync_gateway/blob/e46771e9e3b10cee76e1a896dcd6cb0d7a5359f9/db/blip_sync_context.go#L56-L56

And it is only set to false here https://github.com/couchbase/sync_gateway/blob/e46771e9e3b10cee76e1a896dcd6cb0d7a5359f9/db/active_replicator.go#L245

Reply:
But `arc.config.DeltasEnabled` is a bool so the zero value will match the if condition and turn it off.

See also `DefaultReplicationConfig()` which explicitly sets the default value of `DeltasEnabled` to false in the replication config.

Reply:
How confusing, thanks.